web/app/.editorconfig

```
@@ -0,0 +1,7 @@
+[*{.js,json,ts,tsx,graphql}]
```

Should we consider this projects as "standalone" and this `.editorconfig` file the root one, not accounting for the one at the root of the repo (which actually does not exist yet) ?

Then:

```suggestion
; top-most EditorConfig file
root = true

; Unix-style newlines
[*]
end_of_line = lf
charset = utf-8
insert_final_newline = true
trim_trailing_whitespace = true

[*{.js,json,ts,tsx,graphql}]
```